.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: python
+env:
+  - DJANGO_VERSION="Django>=1.8,<1.9"
+  - DJANGO_VERSION="Django>=1.9,<1.10"
+  - DJANGO_VERSION='https://github.com/django/django/archive/master.tar.gz'
+python:
+  - "2.7"
+  - "3.4"
+before_script:
+  - pip install coverage coveralls
+  - pip install -q "$DJANGO_VERSION"
+  - python setup.py install
+script:
+  - coverage run manage.py test --settings=settings
```

Comment:
When trying to run this locally I get a ``No file to run: 'manage.py'`` error.

Would you mind also adding a tox.ini file?